rwkv.cpp

```diff
@@ -116,17 +116,28 @@ static const ggml_type FORMAT_TYPE_TO_GGML_TYPE[FORMAT_TYPE_COUNT] = {
     GGML_TYPE_Q4_0,
     GGML_TYPE_Q4_1,
     GGML_TYPE_UNKNOWN, // Unused
-    GGML_TYPE_Q4_2,
+    GGML_TYPE_UNKNOWN, // Unused
     GGML_TYPE_UNKNOWN, // Unused
     GGML_TYPE_Q5_0,
     GGML_TYPE_Q5_1,
     GGML_TYPE_Q8_0
 };
 
+static bool is_non_quantized_format_type(int32_t format_type) {
+    return format_type == 0 || format_type == 1;
+}
+
+static bool is_quantized_format_type(int32_t format_type) {
```

Comment:
would recommend using `was` here (`was_quantized_format_type`), because this function returns if the data type ever used to refer to a quantized type in the past, even if the data type is no longer supported

Reply:
Hm, it actually only handles supported formats. For example, it wil return `false` for value of `4`, although in the past it was `Q4_1_O`

Reply:
since this is used for reporting errors when the format is not supported, don't you think it should include old unsupported formats, so that rwkv.cpp knows to report a helpful error ?

Reply:
Check of removed formats (line 495) is just above check of quantized formats (line 499), so this function will not be called for removed formats. Looks like no additional confusion was introduced.